test/resources/compliance_history/policy-uninstall-gk.yaml

```diff
@@ -10,10 +10,9 @@ spec:
         apiVersion: policy.open-cluster-management.io/v1
         kind: ConfigurationPolicy
         metadata:
-          name: compliance-api-uninstall-gk-sub
+          name: compliance-api-uninstall-gk
         spec:
           object-templates-raw: |
-
```

Comment:
(Optional) I personally like this newline because a quirk makes VSCode render the raw template as YAML rather than as a string, but it's fine to remove.